src/styles.ts

```diff
@@ -57,8 +57,8 @@ export const styles: Theme['styles'] = {
   },
   hr: {
     my: 4,
+    border: 'solid',
```

Comment:
```suggestion
    borderStyle: 'solid',
```

Read more here: https://www.w3schools.com/cssref/pr_border-style.asp